cspell.json

```diff
@@ -327,6 +327,8 @@
         "softline",
         "softlines",
         "Sorin",
+        "Sosuke",
+        "sosukesuzuki",
```

Comment:
Can we add patter to ignore `author: ".*？"`？

Reply:
done: 36cce5c
Please review because I'm not familiar with regexp